src/web_video_server.cpp

```diff
@@ -54,7 +54,8 @@ WebVideoServer::WebVideoServer(rclcpp::Node::SharedPtr &nh, rclcpp::Node::Shared
         async_web_server_cpp::HttpReply::stock_reply(async_web_server_cpp::HttpReply::not_found))
 {
   rclcpp::Parameter parameter;
-  if (private_nh->get_parameter("port", parameter)) {
+  nh->declare_parameter("port", rclcpp::PARAMETER_INTEGER);
+  if (nh->get_parameter("port", parameter)) {
```

Comment:
I don't follow this... what does switching from `private_nh` to `nh` mean?

Reply:
I don't know why they have two nodes in the first place - https://github.com/PickNikRobotics/web_video_server/blob/ros2/src/web_video_server.cpp#L381-L382

We set the parameter for node `web_video_server` in the launch file hence I switched getting the parameter value from `private_nh` to `nh`